networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/Eth2TopicHandler.java

```diff
@@ -62,15 +62,20 @@ protected abstract void processMessage(
       final TWrapped message, InternalValidationResult internalValidationResult);
 
   protected ValidationResult handleMessageProcessingError(Throwable err) {
+    final ValidationResult response;
     if (Throwables.getRootCause(err) instanceof DecodingException) {
```

Comment:
BTW I'd fix this check since there are cases when `DecodingException` is not the innermost. E.g.:
```java
    } catch (SSZException e) {
      throw new DecodingException("Failed to deserialize value", e);
    } catch (Exception e) {
      throw new DecodingException("Encountered exception while deserializing value", e);
    }
```

Reply:
good catch!